scripts/update/Updater.php

```diff
@@ -209,11 +209,17 @@ public function update($initialVersion)
 
         if ($this->isVersion('9.0.0')) {
             OntologyUpdater::syncModels();
-
-            $this->getServiceManager()->register(ConsumerService::SERVICE_ID, new ConsumerService());
-            $this->getServiceManager()->register(ProviderService::SERVICE_ID, new ProviderService());
+            // Removed as of 9.2.0 due to autowiring
```

Comment:
Should we keep this as a comment, or simply update the script by skipping from 9.0.0 to 9.1.0?

Reply:
For debugging we should keep the comment, in case there is ever an issue with updating (really useful when you need to update 2 year old instances)